packages/grafana-ui/src/components/Icon/iconBundle.ts

```diff
@@ -171,7 +171,7 @@ export function initIconCache() {
 
   // This function needs to be called after index.js loads to give the
   // application time to modify __webpack_public_path__ with a CDN path
-  const grafanaPublicPath = (window as any).__grafana_public_path__;
+  const grafanaPublicPath = typeof window !== 'undefined' && (window as any).__grafana_public_path__;
```

Comment:
I think this file is autogenerated (as mentioned by the comment at the top of the file 😅 ), so this change will be lost next time we change the icons.

Check https://github.com/grafana/icons/blob/main/cmd/genbundle/main.go#L65

Reply:
There you go https://github.com/grafana/icons/pull/6